docs/workforce-management-framework/topics/con-workflow-step.adoc

```diff
@@ -0,0 +1,63 @@
+= Building your first Workflow Step/Reusing template
+
+A basic knowledge of Angular.js is requried to build a Workflow Step.
+
+== Step Structure
+Steps are divided into to linked catergories the `module` and the `definition`.
+
+=== Module
+Modules can contain one or more angular directives for rendering steps in the portal and mobile applications. 
+{WFM-RC-NameShort} steps use Angular.js directives to build user interface as part of workflows.
+Each step is divided into two types of directive:
+
+- **Edit Section (Form)**
+
+The Edit section is displayed when workflow is executed and allows users to edit information typically this would be a form.
+The Edit section consists of the html template and directive.
+
+- **Preview Section (View)**
+
+The Preview section is displayed as a summary when a workflow has been completed. The Preview section
+consists of the html template and directive.
+
+=== Definition 
+Definition contains the step definition which outlines the properties such as name and decription of a step for the workflow in the portal application.
+Each step requires a definition so the `definition.json` file can contain a single definition or 
+an array of definitions depending on the number of steps that are included in the step package.
+
```

Comment:
Each step requires a definition, the  `definition.json` file can contain a definition of a single step or an array of definitions with the size corresponding to the number of steps in the step package.
?